### PR TITLE
Add Ukrainian; bump version

### PR DESCRIPTION
**Pull request by serhiipalash**

### CODE REVIEW

**Comment by serhiipalash** [comment]
ping @e110c0

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+# 3.1.2
```
Comment (e110c0) [inline CHANGELOG.md:3]: please either change this to 'unreleased' since version might be different

File: entry-node.js [hunk 2]
```diff
@@ -24,6 +24,7 @@ const sv = require('./langs/sv.json');
 const th = require('./langs/th.json');
 const vi = require('./langs/vi.json');
 const zh = require('./langs/zh.json');
+const uk = require('./langs/uk.json');
```
Comment (e110c0) [inline entry-node.js:27]: chore: please add this in alphabetical order

**Review by e110c0 — CHANGES_REQUESTED** [review]

**Comment by serhiipalash** [comment]
@e110c0 I've fixed all issues you mentioned.

**Review by e110c0 — APPROVED** [review]

**Pull request merged**